### ISSUE TITLE
Is this framework still in use?

### ISSUE THREAD

**Issue by karliwalti** [part-cw/LNhealth#1]
I would be happy to help expanding a bit the wiki documentation. 
It is not clear what these sxz files are and  how they can be generated?

**Comment by mgorges** [comment]
@karliwalti yes it is used in production apps. Thanks for your offer to help with the documentation, we appreciate it.

`sxz` are individual LNhealth packages, to be used with the main [LNhealth](https://github.com/part-cw/LNhealth/tree/master/apps/LNhealth) application. They contain the scripts, images, and all other required files to run the package. LambdaNative's [targets/lnhealth/package#L43](https://github.com/part-cw/lambdanative/blob/master/targets/lnhealth/package#L43) creates these zip files in the packaging step.
